pkg/component/clusterautoscaler/bootstrap.go

```diff
@@ -61,6 +61,11 @@ func (b *bootstrapper) Deploy(ctx context.Context) error {
 					Resources: []string{"*"},
 					Verbs:     []string{"create", "delete", "deletecollection", "get", "list", "patch", "update", "watch"},
 				},
+				{
+					APIGroups: []string{"apps"},
+					Resources: []string{"deployments"},
+					Verbs:     []string{"get", "list", "watch"},
+				},
```

Comment:
kindly add a test case if there is for the already existing rules in `bootstrap_test.go`